components/Cards/src/Theming/MDCCard+MaterialTheming.m

```diff
@@ -0,0 +1,83 @@
+// Copyright 2018-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import "MDCCard+MaterialTheming.h"
+
+#import "MaterialCards+ColorThemer.h"
+#import "MaterialCards+ShapeThemer.h"
+
+static const MDCShadowElevation kNormalElevation = 1;
+static const MDCShadowElevation kHighlightedElevation = 4;
+static const CGFloat kBorderWidth = 1;
+
+@implementation MDCCard (MaterialTheming)
+
+#pragma mark - Standard Card
+
+- (void)applyThemeWithScheme:(id<MDCContainerScheming>)scheme {
+  id<MDCColorScheming> colorScheme = scheme.colorScheme;
+  if (!colorScheme) {
+    colorScheme =
+        [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
+  }
+  [self applyThemeWithColorScheme:colorScheme];
+
+  id<MDCShapeScheming> shapeScheme = scheme.shapeScheme;
+  if (!shapeScheme) {
+    shapeScheme = [[MDCShapeScheme alloc] initWithDefaults:MDCShapeSchemeDefaultsMaterial201809];
+  }
+  [self applyThemeWithShapeScheme:shapeScheme];
+
+  [self setShadowElevation:kNormalElevation forState:UIControlStateNormal];
+  [self setShadowElevation:kHighlightedElevation forState:UIControlStateHighlighted];
+  self.interactable = YES;
```

Comment:
This looks unexpected because it’s modifying behavior; is this line intentional? If so, consider documenting why we are setting it (e.g. does the design system require that a card themed in this manner be interactable?)

Reply:
That's a good point. I kept this behavior to match the existing themer. @yarneo perhaps you can shed some light on this?

Reply:
I spoke with @yarneo about it, the line is intentional to achieve the baseline state for an MDCCard, so I'll add a comment